ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json

```diff
@@ -6,48 +6,48 @@
-        "HopeScarves": "[es]Hope Scarves logo",
-        "CancerCouch": "[es]The Cancer Couch logo",
-        "TwistedPink": "[es]Twisted Pink logo",
-        "CierraSisters": "[es]Cierra Sisters logo",
-        "BC": "[es]Breast Cancer Trials logo",
-        "BCAlliance": "[es]Breast Cancer Alliance logo",
-        "TutuProject": "[es]The Tutu Project logo",
-        "MPBCAlliance": "[es]Metaplastic Breast Cancer Global Alliance logo",
-        "LBCA": "[es]Lobular Breast Cancer Alliance logo",
-        "Force": "[es]Facing Our Risk of Cancer Empowered logo",
-        "Shay": "[es]Shay Sharpe's Pink Wishes logo",
-        "MRS": "[es]Metastasis Research Society logo",
-        "Sisters": "[es]Sisters R Us Circle of Survivors logo",
-        "HMN": "[es]Health Ministries Network logo",
-        "Min": "[es]Minority Coalition logo",
-        "MBCCanada": "[es]Metastatic Breast Cancer Canada logo"
+        "MBCN": "logo de Metastatic Breast Cancer Network",
+        "Avon": "logo de Avon Foundation For Women",
+        "Alliance": "logo de MBC Alliance",
+        "LBBC": "logo de Living Beyond Breast Cancer",
+        "IBCRF": "logo de Inflammatory Breast Cancer Foundation",
+        "Young": "logo de Young Survival Coalition",
+        "Share": "logo de Share Cancer Support",
+        "Male": "logo de The Male Breast Cancer Coalition",
+        "Theresa": "logo de Theresa's Research Foundation",
+        "TNBC": "logo de Triple Negative Breast Cancer Foundation",
+        "IBC": "logo de The IBC Network Foundation",
+        "A4BC": "logo de Advocates 4 Breast Cancer",
+        "Metavivor": "logo de Metavivor",
+        "Metup": "logo de Metup",
+        "Tigerlily": "logo de Tigerlily Foundation",
+        "Komen": "logo de Susan G. Komen",
+        "BCRF": "logo de Breast Cancer Research Foundation",
+        "DSLFR": "logo de The Dr. Susan Love Foundation",
+        "BCSM": "logo de The Breast Cancer Social Media",
+        "HopeScarves": "logo de Hope Scarves",
+        "CancerCouch": "logo de The Cancer Couch",
+        "TwistedPink": "logo de Twisted Pink",
+        "CierraSisters": "logo de Cierra Sisters",
+        "BC": "logo de Breast Cancer Trials",
+        "BCAlliance": "logo de Breast Cancer Alliance",
+        "TutuProject": "logo de The Tutu Project",
+        "MPBCAlliance": "logo de Metaplastic Breast Cancer Global Alliance",
+        "LBCA": "logo de Lobular Breast Cancer Alliance",
+        "Force": "logo de Facing Our Risk of Cancer Empowered",
+        "Shay": "logo de Shay Sharpe's Pink Wishes",
+        "MRS": "logo de Metastasis Research Society",
+        "Sisters": "logo de Sisters R Us Circle of Survivors",
+        "HMN": "logo de Health Ministries Network",
+        "Min": "logo de Minority Coalition",
+        "MBCCanada": "logo de Metastatic Breast Cancer Canada"
     },
     "Toolkit": {
         "Common": {
-            "Organization": "Cuenten Conmigo"
+            "Organization": "Count Me In"
```

Comment:
Assuming  we want this in english but want to confirm just incase

Reply:
That is a good question, Sampath. Tania specifically asked me how it was used and I explained that it is used in most place in Angular that would just show the title. So this was her informed choice.